hangman.js

```diff
@@ -0,0 +1,117 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let words = ["chicken", "dog", "bird", "snake", "rabbit", "deer", "cat"];
+  // Pick a random word
+  let word = words[Math.floor(Math.random() * words.length)];
+  // // set tries to 5 for the chances
+  let tries = 5;
+  //tell you how many chances left
+  let livesRemaining = document.getElementById("tries");
+  // Set up the dashes
+  let underScores = [];
+  //variable calling guesses paragraph
+  let progress = document.getElementById("Guesses");
+  //calling random word
+  let win = word.length;
+  // check for letters only
+  let strCheck = /^[a-zA-z]+$/;
+  //letters Guessed
+  // let usedLetters = [];
+
+  for (var i = 0; i < word.length; i++) {
+    underScores[i] = "_";
+  }
+  //assign innerText at answerText
+  let answerText = document.getElementById("dashes");
+  answerText.innerText = underScores.join(" ");
+
+  // form will prevnt loop from occuring when entering a letter.
+  let form = document.querySelector("form");
+  form.addEventListener("submit", event => {
+    // Show the player their progress]
+    answerText.innerText = underScores.join(" ");
+    // Get a guess from the player
+    let input = document.getElementById("input");
+    //grabbing the value entered into the input
+    let guess = input.value;
+    //clearing the input box
+    input.value = "";
+
+    event.preventDefault();
+    // function checkDupes(guess) {
+    //   if (!usedLetters.includes(userInput)) {
+    //     return true;
+    //   } else {
+    //     window.alert("This Letter Was Used Already!");
+    //   }
+    // }
+
+    //pushes guessed letters to letters guessed
+    progress.innerText += " " + guess + ",";
+    //Update the game state with the guess
+    for (let j = 0; j < word.length; j++) {
+      if (!guess.match(strCheck)) {
+        answerText.innerText = "Please Enter a Letter!";
+        break;
+      }
+      //check letters against word
+      if (word[j] === guess) {
+        underScores[j] = guess;
+        answerText.innerText = underScores.join(" ");
+        win--;
+        //checking dupes
+        // usedLetters.push(guess);
+      } //check to see if the letter is in the word
+      else if (word.includes(guess) === false) {
+        tries--;
+        livesRemaining.innerText = "You have " + tries + " left";
+        break;
+      }
+    }
+
+    //add the pictures depending on the lives left
+    let newElement = document.createElement("img");
+    switch (tries) {
```

Comment:
One problem here is that everytime the image is appended, the previous image is never removed. If I guessed a letter right in first try, and type that same letter again, it displays same image multiple times because lives remain the same but images keep getting appended one after the other. Putting removeChild method somewhere will help solving this glitch.